packages/piral-cli-webpack/src/configs/common.ts

```diff
@@ -87,28 +102,53 @@ export function getRules(baseDir: string): Array<RuleSetRule> {
       use: [styleLoader, 'css-loader'],
     },
     {
-      test: /\.m?jsx?$/i,
+      test: /\.m?[jt]sx?$/i,
       use: [
         {
           loader: 'babel-loader',
           options: {
-            presets: ['@babel/preset-env', '@babel/preset-react'],
+            presets: [
+              [
+                '@babel/preset-env',
+                {
+                  targets: {
```

Comment:
This should be all given by the end user.

By default it should just use `preset-env`. If users change, e.g., their package.json then this should be reflected.

We should not force IE9/10 to *any* user and it should be easy to opt-out of IE 11.

Reply:
You are right, moving the configuration to the package.json of the shell application is a more flexible and free configuration method.
I strongly agree to remove the support of ie <11 and there are still some users who have a demand for ie11.